proto/redpanda/api/dataplane/v1alpha1/kafka_connect.proto

```diff
@@ -467,6 +635,24 @@ service KafkaConnectService {
     option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
       summary: "Upsert connector configuration"
       description: "Create a new connector using the given configuration, or update the configuration for an existing connector. Returns information about the connector after the change has been made."
+      responses: {
+        key: "200"
+        value: {
+          description: "Ok"
```

Comment:
```suggestion
          description: "Updated"
```
Does a 200 here always indicate an update? If so, would it make sense to have a more specific description than `OK`, especially since we have multiple success codes?